src/data/actions/comment.js

```diff
@@ -13,16 +13,9 @@ const getComment = comment => ({ type: GET_COMMENT, comment });
 const fetchComment = commentId => (
   (dispatch) => {
     dispatch(startedFetchingComment());
-    return fetch(`https://jsonplaceholder.typicode.com/comments/${commentId}`)
-      .then((response) => {
-        if (response.ok) {
-          return response.json();
-        }
-
-        throw new Error();
-      })
-      .then((data) => {
-        dispatch(getComment(data));
+    return axios.get(`https://jsonplaceholder.typicode.com/comments/${commentId}`)
```

Comment:
Could consider building a `client` module that abstracts this out and takes advantage of [`axios.config`](https://github.com/axios/axios#axiosconfig) so you don't have to keep repeating the base URL and makes the code slightly more readable, easier to test, etc.

```javascript
class Client {
  constructor() {
    // set up axios here
    // Probably a better name than `axiosInstance`
    this.axiosInstance = axios.config({
      // some configuration
    });
  }

  getComment(id) {
    return this.axiosInstance.get(`/comments/${id}`)
  }

  getPosts() {
    return this.axiosInstance.get('/posts');
  }
}
```

Reply:
Thanks @jaebradley! We do have plans to update this cookiecutter to be more in line with edX services, including using our AuthenticatedClient which wraps axios for use with edX services (https://github.com/edx/frontend-auth/tree/master/src/AuthenticatedAPIClient)